tests/integration_tests/lightgbm_tuner_tests/test_alias.py

```diff
@@ -37,3 +38,22 @@ def test_handling_alias_parameter():
     assert 'min_data' not in params
     assert 'min_data_in_leaf' in params
     assert params['min_data_in_leaf'] == 0.2
+
+
+def test_handling_alias_metrics():
+    # type: () -> None
+
+    for alias in ['lambdarank', 'rank_xendcg', 'xendcg', 'xe_ndcg',
+                  'xe_ndcg_mart', 'xendcg_mart', 'ndcg']:
+        lgbm_params = {'metric': alias}
+        _handling_alias_metrics(lgbm_params)
+        assert lgbm_params['metric'] == 'ndcg'
+
+    for alias in ['mean_average_precision', 'map']:
+        lgbm_params = {'metric': alias}
+        _handling_alias_metrics(lgbm_params)
+        assert lgbm_params['metric'] == 'map'
+
+    lgbm_params = {}
+    _handling_alias_metrics(lgbm_params)
+    assert lgbm_params == {}
```

Comment:
These test cases do not check if the metric names are not the alias metrics. How about adding the following test cases to ensure `_handling_alias_metrics` does not change the metric names like `auc` or `rmse`?
```suggestion
    assert lgbm_params == {}

    lgbm_params = {'metric': 'auc'}
    _handling_alias_metrics(lgbm_params)
    assert lgbm_params['metric'] == 'auc'
```

Reply:
Thank you, I added the suggested test cases.